powerdns_sync/lib/Atomia/DNS/PowerDNSDatabase.pm

```diff
@@ -146,6 +146,7 @@ sub add_zone {
 	my $zone_type = shift;
 	my $presigned = shift;
 	my $nsec_type = shift;
+	my $zone_status = shift;
 
```

Comment:
Parameter $zone_status is not always passed when this method is called :https://github.com/atomia/atomiadns/blob/a6e79c2fdd3b37a5c85c2c22ef26f547c4270b5d/powerdns_sync/lib/Atomia/DNS/PowerDNSSyncer.pm#LL408C19-L408C27. The method should work the same when this parameter is not defined.

Reply:
Resolved.